src/nft/components/profile/view/ProfilePage.tsx

```diff
@@ -33,6 +33,23 @@ const ProfilePageColumn = styled(Column)`
   ${ScreenBreakpointsPaddings}
 `
 
+const ProfileHeader = styled.div`
+  margin-right: 20px;
+  margin-left: 20px;
+  font-size: 28px;
+  font-weight: 500;
+  line-height: 38px;
```

Comment:
nit: do we have a common typography we could use here

Reply:
unfortunately no

Reply:
doesn't theme/components/text work here?